gammapy/data/tests/test_observations.py

```diff
@@ -367,12 +367,12 @@ def test_observation_read_single_file_fixed_rad_max():
     assert u.allclose(obs.rad_max.quantity, 0.1414213 * u.deg)
 
 
-@pytest.mark.xfail
 @requires_data()
 class TestObservationChecker:
     def setup_method(self):
         self.data_store = DataStore.from_dir("$GAMMAPY_DATA/cta-1dc/index/gps")
 
+    @pytest.mark.xfail
```

Comment:
So does this mean we have a problem with `observation.check` or specifically with the cta-1dc ?